init-tools.sh

```diff
@@ -123,6 +123,10 @@ if [ ! -e $__INIT_TOOLS_DONE_MARKER ]; then
         echo "ERROR: An error occured when trying to initialize the tools. Please check '$__init_tools_log' for more details."1>&2
         exit 1
     fi
+
+    echo "Updating CLI Nuget Frameworks map..."
+    cp -u $__TOOLRUNTIME_DIR/Nuget.Frameworks.dll $__TOOLRUNTIME_DIR/dotnetcli/sdk/$__DOTNET_TOOLS_VERSION >> $__init_tools_log
```

Comment:
that is what the previous checks for `if [ "$?" != "0" ]; then` checks are doing